utils/update_ansible_galaxy_roles.py

```diff
@@ -0,0 +1,115 @@
+import os
+import subprocess
+import sys
+
+# --- CONFIGURATION ---
+GITHUB_NAMESPACE = "RedHatOfficial"
+RHEL_ROLE_PREFIX = "ansible-role-rhel"
+ROLE_PREFIX = "ansible-role-"
+
+# The token must be exported in your terminal session before running the script:
+# export GALAXY_TOKEN="your_token_here"
+GALAXY_TOKEN = os.getenv("GALAXY_TOKEN")
+
+
+if not GALAXY_TOKEN:
+    print("ERROR: The GALAXY_TOKEN environment variable is not set.")
+    print("Please set it (export GALAXY_TOKEN=\"...\") and try again.")
+    sys.exit(1)
+
+
+def get_role_repositories():
```

Comment:
The other scripts for updating these use PyGitHub, would it make sense to try and use that?